src/pages/MonthlyGood/PersonalizedMonthlyGoodLandingPage.vue

```diff
@@ -1,66 +1,96 @@
 <template>
 	<www-page>
-		<kv-hero class="mg-hero bg-overlay">
-			<template #images>
+		<section>
+			<kv-hero class="mg-hero bg-overlay">
+				<template #images>
+					<kv-contentful-img
+						:contentful-src="heroImage"
+						fallback-format="jpg"
+						:width="1440"
+						:alt="heroImageAlt"
+						:source-sizes="sourceSizes"
+						crop="&fit=fill&f=face"
+					/>
+				</template>
+				<template #overlayContent>
+					<form @submit.prevent.stop="submit" novalidate>
+						<div class="tw-bg-primary-inverse tw-bg-opacity-low
+						md:tw-max-w-sm md:tw-ml-4
+						lg:tw-max-w-md lg:tw-ml-16"
+						>
+							<div class="tw-p-1.5 lg:tw-p-3 tw-mb-1.5">
+								<h2 class="tw-text-primary-inverse tw-mb-2" v-html="heroHeadline"></h2>
+								<p class="tw-text-subhead tw-text-primary-inverse" v-html="heroBody"></p>
+							</div>
+							<fieldset class="tw-mb-2 tw-flex">
+								<div class="tw-inline tw-max-w-xs tw-flex-1 tw-px-1 lg:tw-px-3">
+									<label
+										class="tw-sr-only"
+										:class="{ 'tw-text-danger': $v.$invalid }" :for="'amount-' + componentKey"
+									>
+										Amount
+									</label>
+									<kv-currency-input
+										:id="'amount-' + componentKey"
+										:value="amount"
+										@input="updateAmount"
+										class=""
+									/>
+									<ul class="validation-errors tw-text-danger" v-if="$v.$invalid">
+										<li v-if="!$v.amount.required">
+											Field is required
+										</li>
+										<li v-if="!$v.amount.minValue || !$v.amount.maxValue">
+											Enter an amount of $5-$8,500
+										</li>
+									</ul>
+								</div>
+								<kv-button
+									class="tw-flex-1 tw-mb-2 tw-inline tw-px-1 lg:tw-px-3"
+									type="submit"
+									:state="$v.$invalid ? 'disabled' : ''"
+								>
+									{{ heroPrimaryCtaText }}
+								</kv-button>
+							</fieldset>
+						</div>
+					</form>
+				</template>
+			</kv-hero>
+
+			<personalized-mg-lightbox
+				:show-lightbox="showLightbox"
+				@hide-lightbox="showLightbox = false"
+			/>
+		</section>
+
+		<section class="row md:tw-flex">
+			<div class="tw-block md:tw-hidden">
```

Comment:
Please use flex-box to change the flex order so you can only load 1 image into the page. We do this in dynamicHeroClassic using `tw-order-first` which you may be able toggle by screen size.